site/src/css/custom.css

```diff
@@ -346,22 +337,6 @@ ul.table-of-contents li > ul li {
   --ifm-leading: var(--ifm-paragraph-margin-bottom);
 }
 
-main .container > div > div > div {
```

Comment:
were we being too explicit and leaving things without this styles? I am just worried that div>div>div is not very descriptive to what we are targeting, and it could be easily broke by someone accidentally adding another wrapper, or removing one of those divs... could it not be done with classes?

Reply:
I agree it would have been better to use classes but I'm not sure if it was possible at the time